clustering/kmeans_sample.ipynb

```diff
@@ -0,0 +1,518 @@
+{
```

Comment:
<ul><li>typo:</li><li class="ql-indent-1">Elbow Metho<strong>d</strong></li><li class="ql-indent-1">within-cluster sum of square<strong>s</strong><span class="ql-cursor">﻿</span></li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Women-in-Data-Science-Taipei/2020-WiDS-Taipei-MLCC-Workshop/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>